crates/viewer/re_view_spatial/src/ui_3d.rs

```diff
@@ -97,6 +98,12 @@ fn ease_out(t: f32) -> f32 {
     1. - (1. - t) * (1. - t)
 }
 
+/// Properties stored in blueprint.
+pub struct BlueprintProperties {
```

Comment:
very confusingly named - name implies this is "all the blueprint properties" of _something_.
Also unclear why we need this and don't pass around the camera `ViewProperty` instead of lightly wrap over it if needed

Reply:
Did rename it though